src/cwe_checker_lib/src/analysis/forward_interprocedural_fixpoint.rs

```diff
@@ -267,19 +268,207 @@ pub fn create_computation<'a, T: Context<'a>>(
+    fn mock_project() -> Project {
+        let callee_block = Term {
+            tid: Tid::new("callee_block"),
+            term: Blk {
+                defs: vec![],
+                jmps: vec![Term {
+                    tid: Tid::new("ret"),
+                    term: Jmp::Return(Expression::const_from_i32(42)),
+                }],
+                indirect_jmp_targets: Vec::new(),
+            },
+        };
+        let called_function = Term {
+            tid: Tid::new("called_function"),
+            term: Sub {
+                name: "called_function".to_string(),
+                blocks: vec![callee_block],
+                calling_convention: Some("_stdcall".to_string()),
+            },
+        };
+
+        let caller_block_2 = Term {
+            tid: Tid::new("caller_block_2"),
+            term: Blk {
+                defs: vec![],
+                jmps: vec![],
+                indirect_jmp_targets: Vec::new(),
+            },
+        };
+        let caller_block_1 = Term {
+            tid: Tid::new("caller_block_1"),
+            term: Blk {
+                defs: vec![],
+                jmps: vec![Term {
+                    tid: Tid::new("call"),
+                    term: Jmp::Call {
+                        target: called_function.tid.clone(),
+                        return_: Some(caller_block_2.tid.clone()),
+                    },
+                }],
+                indirect_jmp_targets: Vec::new(),
+            },
+        };
+        let caller_block_2 = Term {
+            tid: Tid::new("caller_block_2"),
+            term: Blk {
+                defs: vec![],
+                jmps: vec![Term {
+                    tid: Tid::new("jmp"),
+                    term: Jmp::Branch(caller_block_1.tid.clone()),
+                }],
+                indirect_jmp_targets: Vec::new(),
+            },
+        };
+        let caller_function = Term {
+            tid: Tid::new("caller_function"),
+            term: Sub {
+                name: "caller_function".to_string(),
+                blocks: vec![caller_block_1, caller_block_2],
+                calling_convention: Some("_stdcall".to_string()),
+            },
+        };
+        let mut project = Project::mock_x64();
+        project
+            .program
+            .term
+            .subs
+            .insert(caller_function.tid.clone(), caller_function.clone());
+
+        project
+            .program
+            .term
+            .subs
+            .insert(called_function.tid.clone(), called_function.clone());
+
+        project
+    }
+
+    #[test]
+    /// Checks if the nodes corresponding to the callee function are first in the worklist.
+    fn check_bottom_up_worklist() {
+        let project = mock_project();
+        let extern_subs = project
+            .program
+            .term
+            .extern_symbols
+            .keys()
+            .cloned()
+            .collect();
```

Comment:
Since you know this list to be empty, you could also just create an empty list for your test.